ci/recipe/meta.yaml

```
@@ -14,14 +14,14 @@ build:
 
 requirements:
   build:
```

```suggestion
   host:
```
New conda uses build for compile toolchains, it's kind of backwards compatible right now, but may as well update it.